tutorials/ar-subs/index.md

```diff
@@ -0,0 +1,795 @@
+
+
+### Install Cloud SDK
+
+1.  Login to the Pi with an SSH connection from your host computer. This way
+you can easily copy & paste commands from this tutorial and linked pages, to
+the Pi. Execute:
+
+        ssh pi@<your-Pi-IP>
+
+1.  On the Pi, follow all the steps [here][cloudsdk] to install and initialize
+    Cloud SDK for Debian systems.
+1.  Check that Cloud SDK is installed and initialized with:
+
+        gcloud info
+
+    Ensure that the `Account` and `Project` properties are set correctly.
+
+[cloudsdk]: https://cloud.google.com/sdk/docs/#deb
+
+
+### Install additional OS packages
+
+1.  Execute the following command to install the required OS package dependencies:
+
+        sudo apt-get update && sudo apt-get install -y git python3-dev python3-pygame python3-venv libatlas-base-dev libasound2-dev python3-pyaudio
+
+
+### Increase console font size
+
+This tutorial uses the HDMI console output of the Raspberry Pi as the main
+display. If the default console font size is too small, you can execute the
+following steps to increase the font size and set it to Terminus 16x32.
+
+1. Execute the following command to run the `dpkg-reconfigure` utility:
+
+        sudo dpkg-reconfigure console-setup
+
+1. Using the up/down arrow keys select `UTF-8`. Then using the right arrow key
+select `OK` and press ENTER.
+1. Using the up/down arrow keys select `Guess optimal character set`. Then
+using the right arrow key select “OK” and press ENTER.
+1. Using the up/down arrow keys select `Terminus`. Using the right arrow key
+select `OK` and press ENTER.
+1. Using the up/down arrow keys select `16×32`. Using the right arrow key
+select `OK` and press ENTER. The console will be refreshed and you will be
+returned to the command prompt with the larger console font.
+
+
+### Suppress some of the ALSA errors
+
+On Raspberry Pi the ALSA sound libraries may output errors when using
+[Pyaudio](https://pypi.org/project/PyAudio/) and the pygame library.
+To suppress some of the ALSA errors when pyaudio starts, execute the following
+steps:
+
+1. Backup the original ALSA config file with:
+
+        sudo cp /usr/share/alsa/alsa.conf /usr/share/alsa/alsa.conf.orig
+
+2. Edit the ALSA config file:
+    1. Search the segment `#  PCM interface`
+    2. Comment out the following lines with: **#** as shown here:
+
+
+        #pcm.front cards.pcm.front
+        #pcm.rear cards.pcm.rear
+        #pcm.center_lfe cards.pcm.center_lfe
+        #pcm.side cards.pcm.side
+        #pcm.surround21 cards.pcm.surround21
+        #pcm.surround40 cards.pcm.surround40
+        #pcm.surround41 cards.pcm.surround41
+        #pcm.surround50 cards.pcm.surround50
+        #pcm.surround51 cards.pcm.surround51
+        #pcm.surround71 cards.pcm.surround71
+        #pcm.iec958 cards.pcm.iec958
+        #pcm.spdif iec958
+        #pcm.hdmi cards.pcm.hdmi
+        #pcm.modem cards.pcm.modem
+        #pcm.phoneline cards.pcm.phoneline
```

Comment:
The indentation is off here. The source code doesn't get displayed correctly

Reply:
Strange it was displayed as a code block in my markdown viewer. Indented one more to the right, let's try this